contrib/babelfishpg_tsql/src/pltsql_utils.c

```diff
@@ -1074,6 +1076,17 @@ is_tsql_any_char_datatype(Oid oid)
 		(*common_utility_plugin_ptr->is_tsql_nvarchar_datatype) (oid);
 }
 
+/* 
+ * tsql data-types that allow syntax varchar(max), nvarchar(max)
+ * varbinary(max) is not allowed as typmod for varbinary if fixed in tdsresponse.
```

Comment:
s/if/is

Why varbinary(max) is handled this way? Can we add test cases for varbinary(max) also?